moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -170,7 +170,6 @@ void ServoCalcs::start()
   auto initial_joint_trajectory = moveit::util::make_shared_from_pool<trajectory_msgs::JointTrajectory>();
 
   initial_joint_trajectory->header.frame_id = parameters_.planning_frame;
-  initial_joint_trajectory->header.stamp = ros::Time::now();
```

Comment:
A comment here about why header.stamp isn't filled in would be great. I assume it's because it defaults to 0

Reply:
Let's set it to 0 explicitly, too. Why not, right?  Something like:

// When a joint_trajectory_controller receives a new command, a stamp of 0 indicates "begin immediately"
// See http://wiki.ros.org/joint_trajectory_controller#Trajectory_replacement
initial_joint_trajectory->header.stamp = 0;

Reply:
Sounds reasonable. I'll change it.